modules/dynamic-operator/Makefile

```diff
@@ -88,28 +90,29 @@ test: manifests generate fmt vet build-docker-compose-plugin envtest ## Run test
 	SERVICE_ACCOUNT="default" \
 	BACKUPS_ENABLED="true" \
 	KL_PLUGIN_DOCKER_COMPOSE_FILE="$(PROJECT_DIR)/config/manager/docker-compose.yaml" \
+	DEPLOYMENT_ID="operator-tests" \
```

Comment:
I don't think that really need. It can be empty for any cases except deploy

Reply:
removed